File: app/src/main/java/com/example/whattodo/model/Coment.java
package com.example.whattodo.model;

public class Coment{

    private String comentari;
    private int puntuacio;
    private String userName;

    public Coment() {}

    public Coment(String userName, String comentari, int puntuacio){
        this.userName = userName;
        this.comentari = comentari;
        this.puntuacio = puntuacio;
    }


    public String getUserName(){ return userName; }

    public String getComentari(){ return comentari; }

    public int getPuntuacio(){ return puntuacio; }

}

File: app/src/main/java/com/example/whattodo/model/Coment.kt
package com.example.whattodo.model

class Coment {
    var comentari: String? = null
        private set
    var puntuacio = 0
        private set
    var userName: String? = null
        private set

    constructor() {}
    constructor(userName: String?, comentari: String?, puntuacio: Int) {
        this.userName = userName
        this.comentari = comentari
        this.puntuacio = puntuacio
    }

}